Movie/MoviesMVP/MovieDescription/Presenter/MovieDescriptionPresenter.swift

```diff
@@ -0,0 +1,70 @@
+// MovieDescriptionPresenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Presenter for MovieDescription ViewController
+final class MovieDescriptionPresenter: MovieDescriptionPresenterProtocol {
+    // MARK: - Private Constants
+
+    private enum Constants {
```

Comment:
```suggestion
    private enum Constants {
```
Если это связано с url это в Нетворк